tests/test1015.py

```diff
@@ -0,0 +1,23 @@
+﻿#Needs: fonts/InterpolateCompatibleCheck.sfd
+#Needs: fonts/InterpolateCompatibleCheck2.sfd
+
+import fontforge
+import sys
+
+f1 = fontforge.open(sys.argv[1])
+f2 = fontforge.open(sys.argv[2])
+
+f1["A"].layers["Fore"].interpolateNewLayer(f2["A"].layers["Fore"], 0.5, True)
```

Comment:
Tests could be more thorough in matching error conditions but that's not a requirement.